test/unit/prompts/Popover.ts

```diff
@@ -1,172 +0,0 @@
-import test from 'ava';
-import sinon, { SinonSandbox } from 'sinon';
-import Popover from '../../../src/popover/Popover';
```

Comment:
there is a typo in the original commit message :( `Popever -> Popover`. We can fix it with reword at the final rebase.

Reply:
Done.